src/components/NonprofitInfoCard.tsx

```diff
@@ -0,0 +1,136 @@
+import { Text, VStack, Heading } from "@chakra-ui/react";
+import { useState, useEffect } from "react";
+import { getNonprofitProject } from "src/actions/Project";
+import { showError } from "src/utils/notifications";
+import { User, Nonprofit } from "src/utils/types";
+
+function NonprofitInfoCard(props: { projectId: string }) {
+  const { projectId } = props;
+
+  const [projectName, setProjectName] = useState("");
+  const [nonprofitName, setNonprofitName] = useState("");
+  const [contactName, setContactName] = useState("");
+  const [contactEmail, setContactEmail] = useState("");
+  const [contactPhone, setContactPhone] = useState("");
+  const [street, setStreet] = useState("");
+  const [city, setCity] = useState("");
+  const [state, setState] = useState("");
+  const [zipCode, setZipCode] = useState("");
+  const [mission, setMission] = useState("");
+  const [appType, setAppType] = useState("");
+  const [isVerified, setIsVerified] = useState(false);
+
+  useEffect(() => {
+    async function preloadFields() {
+      const project = await getNonprofitProject(projectId);
```

Comment:
This is being called twice, once in this component, and again in the application component.  It would be better to make a call for this once in the chapter application component and then pass the data to the two components that need it.  This would avoid a second call and it would avoid an unnecessary call because when the application component is used for the nonprofit application, no calls are needed.